src/controllers/ParagraphStyleController.ts

```diff
@@ -0,0 +1,101 @@
+import { EditorAPI } from '../../types/CommonTypes';
+import { ParagraphStyleUpdate } from '../../types/ParagraphStyleTypes';
+
+/**
+ * The ParagraphStyleController is responsible for all communication regarding paragraph styles.
+ * Methods inside this controller can be called by `window.SDK.paragraphStyle.{method-name}`
+ */
+export class ParagraphStyleController {
```

Comment:
I do not see often the usage of the  `class` in our code. when we need it specifically ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes